api/golang/core/lib/binding_constructors/binding_constructors.go

```diff
@@ -70,12 +70,14 @@ func NewRunStarlarkScriptArgs(serializedString string, serializedParams string,
 	}
 }
 
-func NewRunStarlarkPackageArgs(packageId string, compressedPackage []byte, serializedParams string, dryRun bool, parallelism int32) *kurtosis_core_rpc_api_bindings.RunStarlarkPackageArgs {
+func NewRunStarlarkPackageArgs(packageId string, serializedParams string, dryRun bool, parallelism int32) *kurtosis_core_rpc_api_bindings.RunStarlarkPackageArgs {
 	parallelismCopy := new(int32)
 	*parallelismCopy = parallelism
+	clonePackage := false
 	return &kurtosis_core_rpc_api_bindings.RunStarlarkPackageArgs{
 		PackageId:              packageId,
-		StarlarkPackageContent: &kurtosis_core_rpc_api_bindings.RunStarlarkPackageArgs_Local{Local: compressedPackage},
+		ClonePackage:           &clonePackage,
```

Comment:
We are here taking variable address of something that lives on the stack. Is this fine? Or should we make a defensive copy like a did on `parallelismCopy`?

Reply:
Hmm, it's not on the stack, I instantiate it right above with `clonePackage := true`, so i thiiiiink it fine?

Reply:
Will merge for now, happy to revisit if I'm missing something